src/Data/Nat/Divisibility/Core.agda

```diff
@@ -60,12 +62,18 @@ record _HasNonTrivialDivisorLessThan_ (m n : ℕ) : Set where
     divisor-∣       : divisor ∣ m
 
 ------------------------------------------------------------------------
--- Basic properties
-
-*-pres-∣ : ∀ {m n o p} → o ∣ m → p ∣ n → o * p ∣ m * n
-*-pres-∣ {m} {n} {o} {p} (divides c m≡c*o) (divides d n≡d*p) =
-  divides (c * d) (begin
-    m * n             ≡⟨ cong₂ _*_ m≡c*o n≡d*p ⟩
-    (c * o) * (d * p) ≡⟨ [m*n]*[o*p]≡[m*o]*[n*p] c o d p ⟩
-    (c * d) * (o * p) ∎)
-  where open ≡-Reasoning
+-- DEPRECATED NAMES
+------------------------------------------------------------------------
+-- Please use the new names as continuing support for the old names is
+-- not guaranteed.
+
+-- Version 2.1
+
+*-pres-∣ : o ∣ m → p ∣ n → o * p ∣ m * n
```

Comment:
This is nice! I wonder what happened that resolved this dependency cycle for us. Was it something that you did in this PR?

Reply:
No, in fact it's a knock on consequence of introducing `divides-refl`, albeit one I hadn't noticed, or planned for, at the time of ... whichever PR that was ;-)